CHANGELOG.md

```diff
@@ -2,8 +2,36 @@
 
 ## Unreleased
 
-- Update NHS.UK frontend to 9.4.1
-- Switch to using the `template.njk` that is now included within NHS Frontend ([PR 499](https://github.com/nhsuk/nhsuk-prototype-kit/pull/499))
+### Breaking changes
+
+- Upgrades NHS.UK frontend to 9.4.1 and switches to using a new `prototype-kit-template.njk` within `lib/templates/` which itself uses the `template.njk` that is now included within NHS Frontend ([PR 499](https://github.com/nhsuk/nhsuk-prototype-kit/pull/499))
```

Comment:
This could probably be simplified to something like 'switches to using a new base template to make future upgrades easier'.

Reply:
Good suggestion, thanks.